src/StackAdmin/Azs.AzureBridge.Admin/Tests/src/Override.ps1

```diff
@@ -0,0 +1,2 @@
+
+$global:SkippedTests += "TestDownloadAzsAzureBridgeProduct";
```

Comment:
@deathly809 whats the purpose of this file again? If it's just going to be called in a module's `Common.ps1` file, why don't we just put the functionality from this script into that existing file?

Reply:
We have long haul tests and they want a way to override specific parameters so that they can run these tests on live environments.  This was left in as an example.